avl_tree/AvlTree.java

```diff
@@ -0,0 +1,265 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+import java.util.Random;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Random rand = new Random();
+		int inp = 10000;
+		for (int i = 0; i < inp; i++) {
+			avl.root = avl.insert(avl.root, rand.nextInt(10000));
+		}
+		avl.delNode(avl.root, 1);
+		avl.prntIn(avl.root);

```

Comment:
@Hsankesara That luck can be converted to 100% assurity by just adding 3-4 nodes yourself, and putting that `delNode()` inside a `try-catch` block for some invalid entry. I hope this sounds a good reason for the change?